frontend/src/metabase/questions/containers/MoveToCollection.jsx

```diff
@@ -16,28 +16,29 @@ const mapStateToProps = (state, props) => ({
 
 })
 
-const mapDispatchToProps = {
+const mapDispatchToProps = (dispatch) => ({
     loadCollections,
```

Comment:
Does `loadCollections` still work after changing mapDispatchProps to a function? I don't think the function form auto-binds the actions to dispatch (which would also result in `dispatch(dispatch(...))`)